client/src/components/createCredentialDialog/createCredentialDialog.js

```diff
@@ -313,7 +314,7 @@ class CreateCredentialDialog extends React.Component {
           {this.renderFormLabel('Become Method')}
           <Grid.Col sm={7}>
             <div className="form-split-button">
-              <SplitButton className="form-control" bsStyle="default" title={becomeMethod} id="become-method-button">
+              <SplitButton className="form-control" bsStyle="default" title={becomeMethod} id={helpers.generateId('become-method-button')}>
```

Comment:
Future - generateId should take a prefix string to reduce any chance of duplicates and to give the id some kind of context.